tools/travis/travis_linux_script.sh

```diff
@@ -18,6 +18,17 @@ if ! [ -z ${LINT+x} ]; then
 	cmake ..
 	make cpplint
 	if [ $? -ne 0 ] ; then exit 1 ; fi
+        cd ..
+        cpplint include/*.h
+	if [ $? -ne 0 ] ; then exit 1 ; fi
+        cpplint plugins/*.c
+	if [ $? -ne 0 ] ; then exit 1 ; fi
+        cpplint plugins/*.h
+	if [ $? -ne 0 ] ; then exit 1 ; fi
+        cpplint examples/*.c
+	if [ $? -ne 0 ] ; then exit 1 ; fi
+        cpplint examples/*.h
+	if [ $? -ne 0 ] ; then exit 1 ; fi
```

Comment:
Is there a specific reason why you added the cpp lint calls here?

It would be better to change it here https://github.com/open62541/open62541/pull/1538/commits/4a19333e4d2d6517b7546674f505c1b87eead427#diff-af3b638bc2a3e6c650974192a53c7291R722
so that others can simply run `make cpplint` on their system and it is the same call as on travis.

Reply:
> Is there a specific reason why you added the cpp lint calls here?

No

> It would be better to change it here 4a19333#diff-af3b638bc2a3e6c650974192a53c7291R722
so that others can simply run make cpplint on their system and it is the same call as on travis.

Yes 👍

Reply:
Something like that : https://github.com/StalderT/open62541/pull/3/commits/f1c5e70dd4d58716dfb4ee51ca98d92bdd08aefc ?

The example directory is missing.